### PR TITLE
Remove def software lines and emit versions channel as plural

### PR DESCRIPTION
**Pull request by drpatelh**

### CODE REVIEW

File: modules/unzip/main.nf [hunk 1]
```diff
@@ -24,10 +24,9 @@ process UNZIP {
 
     output:
     path "${archive.baseName}/" , emit: unzipped_archive
-    path "versions.yml"      , emit: version
+    path "versions.yml"      , emit: versions
 
     script:
-    def software = getSoftwareName(task.process)
```
Comment (JoseEspinosa) [inline modules/unzip/main.nf:30]:
```suggestion
```

File: modules/vcftools/main.nf [hunk 2]
```diff
@@ -24,11 +24,11 @@ process VCFTOOLS {
...
     // possible options. Instead we leave that to the user.
     tuple val(meta), path(variant_file)
-    path(bed)
-    path(diff_variant_file)
+    path  bed)
+    path diff_variant_file
 
     output:
-    path("versions.yml") , emit: version
+    path "versions.yml", emit: versions
```
Comment (JoseEspinosa) [inline modules/vcftools/main.nf:32]:
```suggestion
    path "versions.yml",                           emit: versions
```

Comment (JoseEspinosa) [inline modules/vcftools/main.nf:32]:
```suggestion
```

**Review by JoseEspinosa — APPROVED** [review]
Awesome  @drpatelh ! :1st_place_medal: 
Just small comments

**Review by drpatelh — COMMENTED** [review]
Thank you!

**Pull request merged**